From 37573248884ec252165a246df21642c52573f013 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Mon, 8 Apr 2024 16:16:53 +0100
Subject: [PATCH 1020/1085] drivers: mmc: sdhci-brcmstb: bcm2712 supports
 HS400es and clock gating

Enhanced Strobe and clock gating are set in the SDHCI_VENDOR register,
so make the driver aware it needs to do this for this controller.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 drivers/mmc/host/sdhci-brcmstb.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/mmc/host/sdhci-brcmstb.c
+++ b/drivers/mmc/host/sdhci-brcmstb.c
@@ -401,6 +401,8 @@ static const struct brcmstb_match_priv m
 };
 
 static const struct brcmstb_match_priv match_priv_2712 = {
+	.flags = BRCMSTB_MATCH_FLAGS_HAS_CLOCK_GATE,
+	.hs400es = sdhci_brcmstb_hs400es,
 	.cfginit = sdhci_brcmstb_cfginit_2712,
 	.ops = &sdhci_brcmstb_ops_2712,
 };
